scripts/create-associations.js

```diff
@@ -78,15 +78,10 @@ function combineByName(lineItems, creatives) {
   console.log('got all creatives');
 
   lineItems.forEach(function(lineItem) {
-    associations[lineItem.name] = {
-      lineItemId: lineItem.id
-    };
-  });
-
-  creatives.forEach(function(creative) {
-    if (associations[creative.name]) {
-      associations[creative.name].creativeId = creative.id;
-    }
+    associations[lineItem.name] = []
+    creatives.forEach(function(creative) {
+      associations[lineItem.name].push({ lineItemId: lineItem.id, creativeId: creative.id });
+    });
```

Comment:
Oh, now I understand.

Then we probably need two modes for `create-creatives.js` and `create-associations.js`. In the `master`, looks like `create-creatives.js` is taking the (10 line items and 5 creatives = 50 associations) approach. While `create-associations.js` is taking the (10 line items and 10 creatives = 10 associations) approach?